CHANGES.rst

```diff
@@ -7,6 +7,11 @@ Version 2.2.5
 
 *Unreleased*
 
+Improvements
+^^^^^^^^^^^^
+
+- Timetable PDF export: sort posters by Boardnumber
```

Comment:
s/Boardnumber/board number/

Reply:
And add the usual issue/contributor reference:

    (:issue:`4147`, thanks :user:`bpedersen2`)